Makefile

```diff
@@ -12,8 +12,15 @@
 # permissions and limitations under the License.
 
 # Set this to pass additional commandline flags to the go compiler, e.g. "make test EXTRAGOARGS=-v"
-EXTRAGOARGS:=
+CARGO_CACHE_VOLUME_NAME?=cargocache
 DISABLE_ROOT_TESTS?=1
+DOCKER_IMAGE_TAG?=latest
+EXTRAGOARGS:=
+FIRECRACKER_BUILDER_NAME=firecracker-builder
+GOLANG_IMAGE?=golang:1.13-stretch
+FIRECRACKER_BIN=testdata/firecracker-master
+JAILER_BIN=testdata/jailer-master
+FIRECRACKER_TARGET?=x86_64-unknown-linux-musl
```

Comment:
Please remove EXTRAGOARGS and GOLANG_IMAGE. We use Docker only for Firecracker.

Reply:
`EXTRAGOARGS` are needed as we use those to pass go arguments to go commands in the make file.

However, `GOLANG_IMAGE` has been deleted.